boot/bootutil/src/bootutil_misc.c

```diff
@@ -119,6 +119,32 @@ boot_flag_decode(uint8_t flag)
     return BOOT_FLAG_SET;
 }
 
+/**
+ * Determines if a status source table is satisfied by the specified magic
+ * code.
+ *
+ * @param tbl_val               A magic field from a status source table.
+ * @param val                   The magic value in a trailer, encoded as a
+ *                                  BOOT_MAGIC_[...].
+ *
+ * @return                      1 if the two values are compatible;
+ *                              0 otherwise.
+ */
+int
+boot_magic_eq(uint8_t tbl_val, uint8_t val)
```

Comment:
Function name is not descriptive. Maybe `boot_magic_compatible _check`

Reply:
Thanks, I have renamed it as suggested.  To be honest, I did start out with a longer and more descriptive name, but I had a feeling some of these identifiers were getting too long (the switch from 0/1 to primary/secondary definitely contributes to this!).